handlers/handler-bizkeeper/src/main/java/org/apache/servicecomb/bizkeeper/BizkeeperHandler.java

```diff
@@ -82,6 +92,25 @@ public void handle(Invocation invocation, AsyncResponse asyncResp) {
     });
   }
 
+  /**
+   * 使用circuitMarker来记录被熔断的接口
+   * 判断熔断开启或关闭，如果之前没有发送过告警信息才会发送告警
+   */
+  private void eventAlarm(Invocation invocation) {
+    HystrixCommandKey commandKey = CommandKey.toHystrixCommandKey(groupname, invocation);
+    HystrixCircuitBreaker circuitBreaker =
+        HystrixCircuitBreaker.Factory.getInstance(commandKey);
+    String microserviceQualifiedName = invocation.getMicroserviceQualifiedName();
+    if (circuitBreaker != null && circuitBreaker.isOpen() && circuitMarker.get(microserviceQualifiedName) == null) {
+      EventManager.post(new CircutBreakerEvent(invocation, commandKey, this.groupname, Type.OPEN));
+      circuitMarker.put(microserviceQualifiedName, true);
+    } else if (circuitBreaker != null && !circuitBreaker.isOpen()
+        && circuitMarker.get(microserviceQualifiedName) != null) {
+      EventManager.post(new CircutBreakerEvent(invocation, commandKey, this.groupname, Type.CLOSE));
```

Comment:
even circuitMarker no concurrency problem
this logic is not good.

multi threads run if conditions and got true, then all run into post and remove logic.